packages/kit/README.md

```diff
@@ -116,6 +116,36 @@ try {
 }
 ```
 
+### `sendAndConfirmDurableNonceTransactionFactory({rpc, rpcSubscriptions})`
```

Comment:
Added this, since it was missing.

Reply:
nit: probably best if the `sendAndConfirmDurableNonceTransactionFactory` is documented _after_ the `sendAndConfirmTransactionFactory` function on the README. It feels more natural to first read about the "default" send and confirm factory before learning there's also one for durable nonces.

Reply:
Where my A-Z gang at?!

Reply:
Oh my. The most amazing part of this is that this documentation wasn't actually missing. It was there the whole time.